### DISCUSSION TITLE
Aptos token base unit, Octa

### DISCUSSION THREAD

**Discussion by Alexzander-Stone** [aptos-labs/aptos-developer-discussions#78] [Questions] [answered]
Asked by `markkusweird` on Discord (#dev-discussions):

> When we transfer from one account to another (be it a normal account or a resource account)
> We are transfering it in OCTAS, right?
> So if I want to transfer 1 whole Aptos Coin (which is worth about $14 today but that doesn't matter, just for the sake of being on the same page) I need to transfer 100000000 OCTAS here:
> aptos_account::transfer(from, to, amount); 
> the amount will be in OCTAS?
> is that correct? 

**Keywords**
Aptos token, Octa, base unit, conversion, transfer

**Comment by Alexzander-Stone** [comment, marked as answer]
## Revised Questions
There are three questions here:
1. What is the smallest unit of the Aptos token (APT)?
2. How many of this smallest unit make up a single APT?
3. When using `aptos_account::transfer(from, to, amount)` on-chain, what unit of APT is used for the amount argument?

## Answers

### 1. Smallest Unit of the Aptos token
The smallest unit available for APT, is an [**Octa**](https://aptos.dev/reference/glossary/#fungible-unit:~:text=For%20example%2C%20each%20Octa%20(the%20smallest%20unit%20of%20APT)%20is%20a%20fungible%20unit.).

### 2. One Aptos token = ? Octa
[`1` Aptos token is `100,000,000` Octas](https://aptos.dev/reference/glossary/#gas-unit-price:~:text=expressed%20in%20Octa.-,1%20APT%20%3D%2010%5E8%20Octa,-See%20Gas%20and). 
or
`1` Octa = `10^-8` Aptos token.

### 3. What unit does the `amount` argument use, when calling `aptos_account::transfer(from, to, amount)`?
In the [aptos.dev documentation on `aptos_account`](https://aptos.dev/reference/move/?branch=mainnet&page=aptos-framework/doc/aptos_account.md#0x1_aptos_account_transfer), amount is expressed as a `u64` integer. Since we can't have a fractional value on this `u64` amount, we use the smallest unit available for the Aptos token. 
Which is the **Octa**.

**Comment by flaviokicis** [comment]
That is perfect. Thank your for the extremely clear information.
